src/shared/ui/components/BaseDropdown.vue

```diff
@@ -32,26 +32,26 @@ type DropdownItem = {
 }
 
 type Props = {
-  modelValue: string | number,
+  value: string | number,
```

Comment:
Why `value` instead of `moduleValue`? The latter one allows for a shorter `v-model` instead of `v-model:value`, and its just a standard Vue 3 way to define a single model value.

Reply:
moduleValue is not best practice, and is more for those who are used to vue 2.
This view gives us more consistency if we need to add additional v-models.

Reply:
`modelValue` is a **default** prop for `v-model`. It is not a best practice, neither is yours. Your solution is reasonable **if there are other models**. In fact, there are 0 such cases.